src/Commands/ProjectCreateCommand.php

```diff
@@ -426,6 +514,9 @@ function ($state) use ($site_name, $siteDir) {
             ->progressMessage('Install CMS on Pantheon site {site}', ['site' => $site_name])
             ->addCode(
                 function ($state) use ($ci_env, $site_name, $siteDir) {
+                    // Wait for workflow to finish.
+                    // @todo: Is this command working as expected?
+                    $this->passthru("terminus workflow:wait $site_name.dev 'Change database version for an environment'");
```

Comment:
@greg-1-anderson this command doesn't seem to be working as expected. Is there a better way to do this?

If I remove this, it randomly fails because sometimes the site installation is attempted before the db upgrade has been done. Also, waiting for N seconds doesn't seem to be reliable because it sometimes takes longer

Reply:
I think that `pushCodeToPantheon` is already doing a workflow:wait. Maybe the timing or workflow names changed in some way to make this less reliable.